extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/MethodNameParser.java

```diff
@@ -115,7 +115,7 @@ public Result parse(MethodInfo methodInfo) {
         boolean allIgnoreCase = false;
         if (afterByPart.contains(ALL_IGNORE_CASE)) {
             allIgnoreCase = true;
-            afterByPart = afterByPart.replaceAll(ALL_IGNORE_CASE, "");
+            afterByPart = afterByPart.replace(ALL_IGNORE_CASE, "");
```

Comment:
This prevents a regex pattern compilation.

Reply:
Interesting!